.github/workflows/prepare-zephyr/action.yml

```diff
@@ -64,6 +64,11 @@ runs:
         app-path: ${{ inputs.app-path }}
         toolchains: arm-zephyr-eabi:riscv64-zephyr-elf:x86_64-zephyr-elf:arc-zephyr-elf
 
+    - name: Install app python dependencies
+      shell: bash
+      run: |
+        pip install -r ${{ inputs.app-path }}/scripts/requirements.txt
+
```

Comment:
This shouldn't be necessary since pip requirements are install by the upstream setup action.

`west packages pip --install` IIRC

Can you try without?

Reply:
Yes, pip requirements for building *zephyr* are- we have a separate requirements file that includes `tt-flash` and `tt-smi`. I did try this without, here is the run: https://github.com/tenstorrent/tt-zephyr-platforms/actions/runs/14784332664/job/41510027200